ubx_evknorab10_cpuapp_common.dtsi ->  ubx_evknorab10_cpuapp_common.dts

I needed the following rename to build with zepher.
Please check.

ubx_evknorab10_cpuapp_common.dtsi ->  ubx_evknorab10_cpuapp_common.dts

Hiroaki Okoshi

Sure, here it is:

```
vscode ➜ /workspaces/project (2_fix_repo_name_and_container_volume_build) $ west build --build-dir build/blinky_test lib/blinky_app  --pristine --board ubx_mininorab10_nrf5340_cpuapp_ns --no-sysbuild -- -DNCS_TOOLCHAIN_VERSION=NONE -DBOARD_
ROOT=/workspaces/project/third_party/ublox/u-blox-sho-OpenCPU/zephyr
-- west build: making build dir /workspaces/project/build/blinky_test pristine
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /workspaces/project/lib/blinky_app
-- CMake version: 3.22.1
-- Found Python3: /usr/local/bin/python3 (found suitable version "3.12.3", minimum required is "3.8") found components: Interpreter 
-- Cache files will be written to: /home/vscode/.cache/zephyr
-- Zephyr version: 3.5.99 (/toolchain/zephyr/zephyr)
-- Found west (found suitable version "1.2.0", minimum required is "0.14.0")
-- Board: ubx_mininorab10_nrf5340_cpuapp_ns
-- Found host-tools: zephyr 0.16.8 (/toolchain/zephyr-sdk-0.16.8)
-- Found toolchain: zephyr 0.16.8 (/toolchain/zephyr-sdk-0.16.8)
-- Could NOT find Dtc (missing: DTC) (Required is at least version "1.4.6")
-- Found BOARD.dts: /workspaces/project/third_party/ublox/u-blox-sho-OpenCPU/zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpuapp_ns.dts
In file included from <command-line>:
/workspaces/project/third_party/ublox/u-blox-sho-OpenCPU/zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpuapp_ns.dts:10:10: fatal error: ubx_mininorab10_cpuapp_common.dts: No such file or directory
   10 | #include "ubx_mininorab10_cpuapp_common.dts"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
CMake Error at /toolchain/zephyr/zephyr/cmake/modules/extensions.cmake:3901 (message):
  failed to preprocess devicetree files (error code 1):
  /workspaces/project/third_party/ublox/u-blox-sho-OpenCPU/zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpuapp_ns.dts
Call Stack (most recent call first):
  /toolchain/zephyr/zephyr/cmake/modules/dts.cmake:215 (zephyr_dt_preprocess)
  /toolchain/zephyr/zephyr/cmake/modules/zephyr_default.cmake:129 (include)
  /toolchain/zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include)
  /toolchain/zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:92 (include_boilerplate)
  CMakeLists.txt:4 (find_package)


-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: /usr/bin/cmake/bin/cmake -DWEST_PYTHON=/usr/local/bin/python3 -B/workspaces/project/build/blinky_test -GNinja -DBOARD=ubx_mininorab10_nrf5340_cpuapp_ns -DNCS_TOOLCHAIN_VERSION=NONE -DBOARD_ROOT=/workspaces/project/third_party/ublox/u-blox-sho-OpenCPU/zephyr -S/workspaces/project/lib/blinky_app
vscode ➜ /workspaces/project (2_fix_repo_name_and_container_volume_build) $ 
```

Note that the board is configured to `ubx_mininorab10_nrf5340_cpuapp_ns`. When building with the target `ubx_mininorab10_nrf5340_cpuapp` the build succeeds. The error is that line 10 of  `/u-blox-sho-OpenCPU/zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpuapp_ns.dts` is:

```c
#include "ubx_mininorab10_cpuapp_common.dts"
```

Which does not exist. It should be:

```c
#include "ubx_mininorab10_cpuapp_common.dtsi"
```

--- a/zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpuapp_ns.dts
+++ b/zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpuapp_ns.dts
@@ -7,7 +7,7 @@
 
 /dts-v1/;
 #include <nordic/nrf5340_cpuappns_qkaa.dtsi>
-#include "ubx_evknorab10_cpuapp_common.dts"
+#include "ubx_evknorab10_cpuapp_common.dtsi"
 
 / {
 	model = "u-blox EVK-NORA-B10 NRF5340 Application";

--- a/zephyr/boards/arm/ubx_evknorab12_nrf5340/ubx_evknorab12_nrf5340_cpuapp_ns.dts
+++ b/zephyr/boards/arm/ubx_evknorab12_nrf5340/ubx_evknorab12_nrf5340_cpuapp_ns.dts
@@ -7,7 +7,7 @@
 
 /dts-v1/;
 #include <nordic/nrf5340_cpuappns_qkaa.dtsi>
-#include "ubx_evknorab12_cpuapp_common.dts"
+#include "ubx_evknorab12_cpuapp_common.dtsi"
 
 / {
 	model = "u-blox EVK-NORA-B12 NRF5340 Application";

--- a/zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpuapp_ns.dts
+++ b/zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpuapp_ns.dts
@@ -7,7 +7,7 @@
 
 /dts-v1/;
 #include <nordic/nrf5340_cpuappns_qkaa.dtsi>
-#include "ubx_mininorab10_cpuapp_common.dts"
+#include "ubx_mininorab10_cpuapp_common.dtsi"
 
 / {
 	model = "u-blox MINI-NORA-B10 NRF5340 Application";

--- a/zephyr/boards/arm/ubx_mininorab12_nrf5340/ubx_mininorab12_nrf5340_cpuapp_ns.dts
+++ b/zephyr/boards/arm/ubx_mininorab12_nrf5340/ubx_mininorab12_nrf5340_cpuapp_ns.dts
@@ -7,7 +7,7 @@
 
 /dts-v1/;
 #include <nordic/nrf5340_cpuappns_qkaa.dtsi>
-#include "ubx_mininorab12_cpuapp_common.dts"
+#include "ubx_mininorab12_cpuapp_common.dtsi"
 
 / {
 	model = "u-blox MINI-NORA-B12 NRF5340 Application";

